app/assets/javascripts/modules/map/leaflet-plugins/granule_grid_layer.js.coffee

```diff
@@ -37,12 +38,22 @@ ns.GranuleGridLayer = do (
 
   class GranuleGridLayer extends L.GridLayer
     initialize: (@collection, color, @multiOptions) ->
+      console.warn 'collection', @collection
       if @collection.granuleDatasource()
         @granules = @collection.granuleDatasource().data()
       else
         @_datasourceSubscription = @collection.granuleDatasource.subscribe(@_subscribe)
 
-      @color = color ? '#25c85b';
+      # If on the search page, the color should always be green.
+      if page.current.page() == 'search'
+        color = '#25c85b'
```

Comment:
Feels like we may want to use a class, or store these colors in variables or configs some how, hard-coding hex values seems like a bad idea.

Reply:
Totally agree here. I will work something up.